# Vehicle model, mass and inertia matrix

uint64 timestamp		# time since system start (microseconds)

float32 mass			# vehicle mass
float32[9] inertia		# vehicle inertia matrix
